lib/tasks/technical_metadata.rake

```diff
@@ -2,8 +2,10 @@
 
 namespace :techmd do
   desc 'Generate technical metadata (synchronously) from druid + filepaths'
-  task :generate, %i[druid filepaths] => :environment do |_task, args|
-    errors = TechnicalMetadataGenerator.generate(druid: args[:druid], filepaths: args[:filepaths].split(' '))
+  task :generate, %i[druid filepaths force] => :environment do |_task, args|
+    errors = TechnicalMetadataGenerator.generate(druid: args[:druid],
+                                                 filepaths: args[:filepaths].split(' '),
+                                                 force: args[:force] == 'true')
```

Comment:
Does it matter that you are using a string value for `'true'` and not a boolean?

Reply:
I just verified this with testing, but `args[:force]` is a string, not a boolean. Thus, `args[:force] == 'true'` is just converting the string to a boolean.